src/strongbox/ui/jfx.clj

```diff
@@ -1123,6 +1123,21 @@
                                              {:graphic (href-to-hyperlink row)})}
                   :cell-value-factory identity}
          :source-id {:min-width 60 :pref-width 150}
+         :source-map-list {:cell-factory {:fx/cell-type :tree-table-cell
+                                          :describe (fn [row]
+                                                      (let [urls (for [source-map (:source-map-list row)
+                                                                       :let [url (utils/addon-source-map-to-url row source-map)]
```

Comment:
`utils/addon-source-map-to-url` is replicating `*/make-url`logic that already exists